.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,161 @@
+# Contributing to WasmEdge
+
+
+## Welcome
+
+[**WasmEdge**](https://github.com/WasmEdge/WasmEdge) (previously known as SSVM) is a high-performance WebAssembly (Wasm) VM optimized for Edge Computing, including Edge Clouds and Software Defined Vehicles.
+
+WasmEdge is developed in the open, and is constantly being improved by our **users, contributors, and maintainers**. It is because of you that we can bring great software to the community.
+
+This guide provides information on filing issues and guidelines for open source contributors. **Please leave comments / suggestions if you find something is missing or incorrect.**
+
+If you are looking for ideas for contribution, [here is a wish list](wish_list.md) of items we'd like to get some help with!
+
+> The WasmEdge project adopts [DCO](https://community.openhab.org/t/dco-check-signing-off-with-github-web-editor-explanation/83330) to manage all contributions. Please make sure you add your `sign-off-statement` throught the `-s` flag or the GitHub Web UI before committing the pull request message.
+
+## Getting Started
+
+
+### Fork Repository
+
+Fork the WasmEdge repository on GitHub to your personal account.
+
+
+```
+$ git clone git@github.com:WasmEdge/WasmEdge.git
+$ cd WasmEdge
+```
+
+Notes: Note the WasmEdge team builds lots of extensions of Server-side WebAssembly, see [TensorFlow](https://github.com/second-state/SSVM-tensorflow), [Storage](https://github.com/second-state/SSVM-storage), [Command interface](https://github.com/second-state/ssvm_process_interface), [Ethereum](https://github.com/second-state/SSVM-evmc), [Substrate](https://github.com/second-state/substrate-ssvm-node). If you want to contribue the extensions, please go to those repositories.
```

Comment:
TensorFlow: https://github.com/second-state/WasmEdge-tensorflow
Storage: https://github.com/second-state/WasmEdge-storage
Command interface: https://github.com/second-state/wasmedge_process_interface
Ethereum: https://github.com/second-state/WasmEdge-evmc

Reply:
This should still be fixed but in a different file `docs/CONTRIBUTING.md`.